aws-blog-event-driven-batch-analytics/README.md

```diff
@@ -0,0 +1,137 @@
+# Building Event Driven Batch Analytics on AWS
+
+## Overview
+
+This repository contains the code that supports the [AWS Big Data Blog Post](https://blogs.aws.amazon.com/bigdata/)
+
+### Usecase Description
+Yummy Foods has franchise stores all over the country. These franchise stores run on hetrogenous platforms and they submit cumulative transaction files to Yummy Foods at varuious cadence levels through out the day in tab delimited .tdf format. Some of these franchise stores due to their system limitations ocassionally send additional data starting with characters such as “----“.
```

Comment:
We should cite that Yummy Foods is a hypothetical entity, rather than implying it is a real customer. Need to change spelling of 'varuious'. Maybe cite why they receive files at different cadences through the day? I would change the last sentence to 'Due to a limitation in some of their systems, some franchise stores  ocassionally send additional data starting with characters such as “----“'